inference-contract/test/script_test.sh

```diff
@@ -150,6 +150,8 @@ try ${PDO_HOME}/contracts/inference/scripts/ss_start.sh -c -o ${PDO_HOME}/logs -
     --config-dir ${PDO_HOME}/etc/contracts \
     --identity guardian_sservice
 
+sleep 3
```

Comment:
in ccf we wait for a file to be created. in pdo we curl a url found in the configuration file. i don't think either approach would work here. there is no guarantee that the db file (the obvious example) was removed between runs and the configuration file has several urls in it that make the grep approach "brittle" at best. maybe a better way for this service AND for the ones in pdo is to explicitly write a URL into a file (like what we do with the pid) and then use that.

Reply:
@g2flyer curious about your thoughts on https://github.com/hyperledger-labs/private-data-objects/issues/476. This would be a clean way to address your concerns as well.